rundeckapp/grails-spa/packages/ui-trellis/src/app/components/job/options/OptionEdit.vue

```diff
@@ -0,0 +1,1074 @@
+<template>
+  <div class="optEditForm">
+    <div class="alert alert-danger" v-if="error">
+      {{ error }}
+    </div>
+    <div class="row" v-if="newOption">
+      <div class="col-sm-12">
+        <span class="h4">{{ $t("add.new.option") }}</span>
+      </div>
+    </div>
+
+    <!-- TODO restructure the following into subcomponents -->
```

Comment:
I kind of think maybe it is not worth the effort right now, since we will be refactoring all the UI with a component library eventually, and I think things like a set of Form inputs maybe can be redesigned to use a schema based approach to avoid writing a lot of code like this anyway